### PR TITLE
Add doc about php-cs-fixer and the git pre-commit

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: src/content/1.7/development/coding-standards/_index.md [hunk 1]
```diff
@@ -29,6 +29,15 @@ PHP files MUST follow the [PSR-2 standard](https://www.php-fig.org/psr/psr-2/) a
 Altough [Yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions) are suggested, they are not enforced.
 {{% /notice %}}
 
+[PHP CS Fixer](https://cs.sensiolabs.org/) has been configured for the PrestaShop project to help developers to comply with these conventions.
+
+You can run it using the following command:
+```bash
+php bin/php-cs-fixer fix
```
Comment (PierreRambaud) [inline src/content/1.7/development/coding-standards/_index.md]: `php ./vendor/bin/php-cs-fixer fix` instead

File: src/content/1.7/development/coding-standards/_index.md [hunk 2]
```diff
@@ -29,6 +29,15 @@ PHP files MUST follow the [PSR-2 standard](https://www.php-fig.org/psr/psr-2/) a
...
 {{% /notice %}}
 
+[PHP CS Fixer](https://cs.sensiolabs.org/) has been configured for the PrestaShop project to help developers to comply with these conventions.
+
+You can run it using the following command:
+```bash
+php ./vendor/bin/php-cs-fixer fix
+```
+
+And you can also use the provided [git pre-commit](https://github.com/PrestaShop/PrestaShop/tree/develop/.github/contrib) sample in order to make sure you never forget to make your code compliant !
```
Comment (mickaelandrieu) [inline src/content/1.7/development/coding-standards/_index.md]: How about the Composer command which install the git hook? https://github.com/PrestaShop/PrestaShop/blob/develop/composer.json#L176

**Comment by matks** [comment]
@mickaelandrieu I did not notice the Composer command first.

Now I have tried it. It does not work for me. When I look at the code it looks buggy:
https://github.com/PrestaShop/PrestaShop/blob/develop/.github/contrib/install.php#L33
`file_exists('pre-commit')` will never return `true` ? it should be `file_exists('.git/hooks/pre-commit')` or do I miss something ?

**Comment by mickaelandrieu** [comment]
So we don't document it,
this issue report may interest @PierreRambaud ;)

Thanks for the contribution :)

**Pull request merged**